pkg/planner/core/find_best_task.go

```diff
@@ -3341,3 +3341,94 @@ func validateTableSamplePlan(ds *logicalop.DataSource, t base.Task, err error) e
 	}
 	return nil
 }
+
+// mockLogicalPlan4Test is a LogicalPlan which is used for unit test.
```

Comment:
this struct is for test, is it ok to move it here?
and why we are doing this?

Reply:
Becasue we has removed `exhaustPhysicalPlan` in ine logical interface. so we need to exhaust mockLogicalPlan4Test physical op by the hand. so it logical is in the ```exhaustPhysicalPlans``` function.